Gordon360/Services/NewsService.cs

```diff
@@ -272,6 +272,27 @@ public StudentNewsViewModel EditPosting(int newsID, StudentNewsUploadViewModel n
             return newsItem;
         }
 
+        public StudentNewsViewModel AlterPostAcceptStatus(int newsID, bool isAccepted)
```

Comment:
We should either use `Accepted` or `Approved`, but not both. The database object itself has `Accepted` as the column name, so it is probably better to stick with that. Could you update it throughout your code changes?